### ISSUE TITLE
ops.reduce_*/ops.vector_norm multiple reduction dim

### ISSUE THREAD

**Issue by hlky** [hlky/AITemplate#8]
ops.reduce_*/ops.vector_norm currently only support a single reduction dim

[torch.sum](https://pytorch.org/docs/stable/generated/torch.sum.html)
[torch.mean](https://pytorch.org/docs/stable/generated/torch.mean.html)
[torch.linalg.vector_norm](https://pytorch.org/docs/stable/generated/torch.linalg.vector_norm.html)

**Comment by hlky** [comment]
Low priority. Workaround found:

```python
ops.vector_norm(dim=(1, 2), keepdim=True)(x)
# ->
ops.unsqueeze(1)(
    ops.vector_norm(dim=1, keepdim=True)(ops.vector_norm(dim=1)(x))
)
```
